Build error with embroider

Trying to set up embroider - I ran into this build issue:

```
Build Error (PackagerRunner) in ../../node_modules/@ember/test-waiters/index.ts

Module parse failed: Unexpected token (1:7)
File was processed with these loaders:
 * ...node_modules/@embroider/babel-loader-8/index.js
You may need an additional loader to handle the result of these loaders.
> export type { WaiterName, Token, Primitive, Waiter, TestWaiter, TestWaiterDebugInfo, PendingWaiterState } from './types';
```

---

Is this maybe because of the `main` field pointing to the index.ts instead of index.js?

https://github.com/emberjs/ember-test-waiters/blob/master/package.json#L14

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package

I'm getting the same error
``` 
ERROR in ../../node_modules/@ember/test-helpers/-internal/debug-info.js 19:0-60
Module not found: Error: Can't resolve '@ember/test-waiters' in '$TMPDIR/embroider/bfc6a7/node_modules/@ember/test-helpers/-internal'
 @ ./assets/test.js 46:9-82
```

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "Steve Calvert <steve.calvert@gmail.com>",
     "Robert Jackson <me@rwjblue.com>"
   ],
-  "main": "addon/@ember/test-waiters/index.ts",
+  "main": "addon/@ember/test-waiters/index.js",
   "types": "index.d.ts",
   "directories": {
     "doc": "doc",

